### PR TITLE
no-jira: Removing unused terraform variables

### PR DESCRIPTION
**Pull request by barbacbd**
** Removed unused teraform variables in aws terraform variables file.
** Removed unsued variables in gcp terraform variables file.
** Removed unsued variables in azure terraform variables file.

### CODE REVIEW

File: pkg/asset/cluster/tfvars/tfvars.go [hunk 1]
```diff
@@ -396,18 +396,6 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
...
-			// Due to the SAS created in Terraform to limit access to bootstrap ignition, we cannot know the URL in advance.
-			// Instead, we will pass a placeholder string in the ignition to be replaced in TF once the value is known.
-			bootstrapIgnURLPlaceholder = "BOOTSTRAP_IGNITION_URL_PLACEHOLDER"
-			shim, err := bootstrap.GenerateIgnitionShimWithCertBundleAndProxy(bootstrapIgnURLPlaceholder, installConfig.Config.AdditionalTrustBundle, installConfig.Config.Proxy)
-			if err != nil {
-				return errors.Wrap(err, "failed to create stub Ignition config for bootstrap")
-			}
-			bootstrapIgnStub = string(shim)
-		}
-
```
Comment (r4f4) [inline pkg/asset/cluster/tfvars/tfvars.go:410]: I think this breaks azure stack.

**Review by r4f4 — CHANGES_REQUESTED** [review]
The variables from `data/data/azure/variables-azure.tf` regarding azurestack cannot be removed because the config file is shared by both azure and azurestack.

**Review by r4f4 — COMMENTED** [review]
LGTM but I don't know anything about the `*_config_version` variables. Are they some part of an idea that never took off?

It's unfortunate that `tf-fmt` is not behaving consistently: in some instances it aligns the `=` and in others it doesn't :man_shrugging:

**Comment by patrickdillon** [comment]
> It's unfortunate that `tf-fmt` is not behaving consistently: in some instances it aligns the `=` and in others it doesn't 🤷‍♂️

I think it depends on how big the gap is. it will only align if the strings are somewhat similar in length

**Comment by r4f4** [comment]
> > It's unfortunate that `tf-fmt` is not behaving consistently: in some instances it aligns the `=` and in others it doesn't 🤷‍♂️
> 
> I think it depends on how big the gap is. it will only align if the strings are somewhat similar in length

TIL.

**Pull request merged**